h/static/scripts/test/drafts-test.js

```diff
@@ -54,9 +54,9 @@ describe('drafts', function () {
 
   describe('#unsaved', function () {
     it('should return drafts for unsaved annotations', function () {
-      var model = {};
+      var model = {$$tag: 'local-tag', id: undefined};
       drafts.update(model, {text: 'bar'});
-      assert.deepEqual(drafts.unsaved(), [model]);
+      assert.match(drafts.unsaved(), sinon.match([model]));
```

Comment:
`sinon.assert.match` doesn't seems to be documented, and it's also not needed if you use a matcher in the args.